Subject: hyperv: use 3.4 as LIC version string
From: <ohering@suse.de>
Patch-mainline: never
References: bnc#822431

While we have LIS 3.4 in the kernel we should mark it as such.

---
 include/linux/hyperv.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.0-SLE11-SP2/include/linux/hyperv.h
===================================================================
--- linux-3.0-SLE11-SP2.orig/include/linux/hyperv.h
+++ linux-3.0-SLE11-SP2/include/linux/hyperv.h
@@ -403,7 +403,7 @@ hv_get_ringbuffer_availbytes(struct hv_r
  *
  * 3.1 - Added completed hv_utils driver. Shutdown/Heartbeat/Timesync
  */
-#define HV_DRV_VERSION           "3.1"
+#define HV_DRV_VERSION           "3.4"
 
 
 /*
